inc/languages/english/usercp.lang.php

```diff
@@ -211,6 +211,8 @@
 $l['contact_field_icq'] = "ICQ Number:";
 $l['contact_field_skype'] = "Skype ID:";
 $l['contact_field_google'] = "Google Hangouts ID:";
+$l['contact_field_error'] = "Sorry, but we cannot change your contact information as the ID specified is too long. You can try to use a shorter ID number which is less than 75 characters long. This error will be caused by either the Google Hangouts ID or Skype ID.";
```

Comment:
This seems a bit long, ideally error messages should be short and sweet while still being descriptive IMO.

```suggestion
$l['contact_field_error'] = "Sorry, but we cannot change your contact information as the ID specified is too long. Google Hangouts IDs and Skype IDs can be up to 75 characters long.";
```